packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/Camera2CameraInfoHostApiImpl.java

```diff
@@ -86,6 +95,7 @@ public Camera2CameraInfoHostApiImpl(
 
   @Override
   @NonNull
+  @OptIn(markerClass = androidx.camera.camera2.interop.ExperimentalCamera2Interop.class)
```

Comment:
I don't think this is needed here?

Reply:
Removed.

Reply:
I am getting this from the linter (I am using Android Studio):

This declaration is opt-in and its usage should be marked with  '@androidx.camera.camera2.interop.ExperimentalCamera2Interop' or '@OptIn(markerClass = androidx.camera.camera2.interop.ExperimentalCamera2Interop.class)' More... (Ctrl+F1) 
Inspection info: This API has been flagged as opt-in with error-level severity.  Any declaration annotated with this marker is considered part of an unstable or otherwise non-standard API surface and its call sites should accept the opt-in aspect of it either by using @OptIn or by being annotated with that marker themselves, effectively causing further propagation of the opt-in aspect.  Issue id: UnsafeOptInUsageError Vendor: Android Open Source Project (jetified-annotation-experimental-1.1.0) Identifier: jetified-annotation-experimental-1.1.0 Feedback: https://issuetracker.google.com/issues/new?component=192731

I see 3 options here:
- Remove the annotation and accept the warning (this is annoying).
- Keep the annotation.
- Add a linter suppression:
```java
@SuppressLint("UnsafeOptInUsageError") final Camera2CameraInfo camera2CameraInfo = proxy.createFrom(cameraInfo);
```
Despite the linter complaining it doesn't seem to actually make any difference, though.

Any preference or other suggestion?

Reply:
Indeed, the @OptIn I removed from line 57 was redundant, because its class, Camera2CameraInfoProxy, did have that annotation. This method, however, createFrom(...), belongs to Camera2CameraInfoHostApiImpl, which didn't have the annotation and if I remove it, the linter complains. So, for consistency I am moving the @OptIn to Camera2CameraInfoHostApiImpl, and removing all other @OptIns.